src/internal/packager/sbom/tools.go

```diff
@@ -46,3 +49,41 @@ func WriteSBOMFiles(sbomViewFiles []string) error {
 
 	return nil
 }
+
+// ViewSBOMFiles opens a browser to view the SBOM files and pauses for user input
+func ViewSBOMFiles(tmp types.TempPaths) {
+	sbomViewFiles, _ := filepath.Glob(filepath.Join(tmp.Sboms, "sbom-viewer-*"))
+
+	if len(sbomViewFiles) > 0 {
+		link := sbomViewFiles[0]
+		msg := fmt.Sprintf("This package has %d images with software bill-of-materials (SBOM) included. If your browser did not open automatically you can copy and paste this file location into your browser address bar to view them: %s\n\n", len(sbomViewFiles), link)
+		message.Note(msg)
+
+		if err := utils.ExecLaunchURL(link); err != nil {
+			message.Debug(err)
+		}
+
+		// Use survey.Input to hang until user input
+		var value string
+		prompt := &survey.Input{
+			Message: "Hit the 'enter' key when you are done viewing the SBOM files",
+			Default: "",
+		}
+		_ = survey.AskOne(prompt, &value)
+	} else {
+		message.Note("There were no images with software bill-of-materials (SBOM) included.")
+	}
+}
+
+// OutputSBOMFiles outputs the sbom files into a specified directory
+func OutputSBOMFiles(tmp types.TempPaths, outputDir string) error {
+	if err := os.RemoveAll(outputDir); err != nil {
```

Comment:
My thought was that may get confusing since it would just dump all sboms into a single folder and you wouldn't know which sboms came from where anymore.  We could create sub directories for them potentially if we wanted (though we would need to be careful about name collisions there too).